### ISSUE TITLE
Unity 2019.x produces build errors for iOS related to usage of TargetGuidByName

### ISSUE THREAD

**Issue by manelizzard** [mopub/mopub-unity-sdk#150]
iOS won't build properly due to `project.TargetGuidByName("Unity-iPhone");` in `MoPubPostBuildiOS`

- [x] I am submitting a bug report for existing functionality
- [x] I visited https://developers.mopub.com/ and found no answer
- [x] I checked https://twittercommunity.com/c/advertiser-api/mopub to make sure that this issue has not already been filed
- [x] I checked to make sure that this issue has not already been filed

#### MoPub SDK Version:
5.15.0
#### Device model and OS Version:
N/A
#### Unity Version:
2019.4.13f1
#### Ad Unit IDs used in reproducing the issue:
N/A
#### Steps to reproduce the behavior:
Try to build an iOS project from a Unity game/app. XCode project will be generated but not Podfile nor xcworkspace

#### Expected behavior:
XCode project generated properly with Podfile and *.xcworkspace

#### Observed behavior:
Error building iOS: `Exception: Calling TargetGuidByName with name='Unity-iPhone' is deprecated. There are two targets now, call GetUnityMainTargetGuid() - for app or GetUnityFrameworkTargetGuid() - for source/plugins to get Guid instead.`

#### Evidence:

<img width="1675" alt="Captura de pantalla 2020-11-20 a las 21 36 28" src="https://user-images.githubusercontent.com/895589/99847500-7988a400-2b78-11eb-9d39-182223d5c64f.png">

#### Proposed fix:

Replace `var target = project.TargetGuidByName("Unity-iPhone");` with
```
var target;
#if UNITY_2019_3_OR_NEWER
    target = project.GetUnityMainTargetGuid();
#else
    target = project.TargetGuidByName("Unity-iPhone");
#endif
```
in `MoPubPostBuildiOS.cs:42`

**Comment by chauduyphanvu** [comment]
@manelizzard, thanks! I was able to reproduce this issue and have let the rest of the team know. We'll share an update here. Stay tuned!

Vu Chau
MoPub Team

**Comment by chauduyphanvu** [comment]
This issue has been addressed internally. A fix will be deployed in the upcoming SDK release.

**Comment by vincent-savysoda** [comment]
It's February now...where is this fix?

**Comment by vincent-savysoda** [comment]
It's Feb 2020 now, where is this fix?

**Comment by chauduyphanvu** [comment]
@vincent-savysoda, we expect a release to be ready by the end of the month. I'll update this thread.

Thank you!

**Comment by wacamoe** [comment]
Hi @vincent-savysoda - this has been addressed in the [5.16.0](https://github.com/mopub/mopub-unity-sdk/releases/tag/v5.16.0) release. 

Thank you again for your report.
